client/add/add.ctrl.js

```diff
@@ -139,11 +139,19 @@ module.exports = class Add {
         );
         return lines;
       })
-      .then(lines => this.boxes.forEach((box) => {
+      .then(lines => {
```

Comment:
Having thought it through, this probably makes the most sense to do. We can get the boxes by id on initial data and everytime we add a new box. That won't be much, because when we get the new box, we know the id and can just insert it. All good.